lookout/style/format/__main__.py

```diff
@@ -6,7 +6,7 @@
 from lookout.core.cmdline import ArgumentDefaultsHelpFormatterNoNone
 from lookout.core.slogging import setup as setup_slogging
 from lookout.style.format.quality_report import quality_report
-from lookout.style.format.robustness import style_robustness_report
+from lookout.style.format.robustness import style_robustness_report, plot_pr_curve
```

Comment:
wrong order of import. It's actually super strange that travis passes the style check considering that. Same thing in the test file.